getting_started.md

```diff
@@ -0,0 +1,62 @@
+# Building, Generating, and Uploading to S3
+## Installation Requirements
+### MATLAB   
+1. MATLAB                    - Required  
+2. Simulink                  - Required  
+3. HDL Coder                 - Required  
+4. DSP System Toolbox        - Strongly Suggested, Required for Example Designs  
+5. Fixed-Point Designer      - Strongly Suggested, Required for Example Designs  
```

Comment:
I wonder if Fixed-Point Designer is actually required. I guess if people could make designs without fixed-point objects in Simulink, it's not strictly needed, though in practice it probably is.

Reply:
Updated MATLAB Requirements - https://www.mathworks.com/support/requirements/hdl-coder.html
Commit: e234329